tad/api/routes/tasks.py

```diff
@@ -0,0 +1,48 @@
+from typing import Annotated, Any
+
+from fastapi import APIRouter, Depends, Request, status
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from tad.models.task import MoveTask
+from tad.services.tasks import TasksService
+
+router = APIRouter()
+templates = Jinja2Templates(directory="tad/site/templates")
+
+
+@router.post("/move", response_class=HTMLResponse)
```

Comment:
that reponse type should not be needed since it is de default. (see main.py in  the root)